pkg/playground/test.js

```diff
@@ -132,4 +132,25 @@ document.addEventListener("DOMContentLoaded", () => {
 
     cockpit.addEventListener("visibilitychange", show_hidden);
     show_hidden();
+
+    const fsreplace_btn = document.getElementById("fsreplace1-create");
+    const fsreplace_error = document.getElementById("fsreplace1-error");
+    fsreplace_btn.addEventListener("click", e => {
+        fsreplace_btn.disabled = true;
+        fsreplace_error.textContent = '';
+        const filename = document.getElementById("fsreplace1-filename").value;
+        const content = document.getElementById("fsreplace1-content").value;
+        const use_tag = document.getElementById("fsreplace1-append").checked;
+        const file = cockpit.file(filename, { superuser: "try" });
```

Comment:
It would be useful to test it with both superuser and as normal user. It's not clear at all whether changing SELinux context works as user.